pkg/power/components/power.go

```diff
@@ -56,8 +57,13 @@ func InitPowerImpl() {
 			klog.V(1).Infoln("use MSR to obtain power")
 			powerImpl = msrImpl
 		} else {
-			klog.V(1).Infoln("Not able to obtain power, use estimate method")
-			powerImpl = estimateImpl
+			if apmXgeneSysfsImpl.IsSystemCollectionSupported() {
```

Comment:
maybe need check whether github action has ARM/latest kernel support later on to test this branch?

Reply:
GHA doesn't have ARM, this has to be tested on a dedicated platform